assets/scripts/palette/__tests__/UndoRedo.test.js

```diff
@@ -1,98 +1,103 @@
 /* eslint-env jest */
 import React from 'react'
-import { mountWithIntl as mount } from '../../../../test/helpers/intl-enzyme-test-helper.js'
-import { UndoRedo } from '../UndoRedo'
-import { isUndoAvailable, isRedoAvailable } from '../../streets/undo_stack'
-
-jest.mock('../../streets/undo_stack')
-jest.mock('../../store/actions/undo', () => ({
-  undo: jest.fn(),
-  redo: jest.fn()
+import { cleanup, fireEvent } from '@testing-library/react'
+import { renderWithReduxAndIntl } from '../../../../test/helpers/render'
+import UndoRedo from '../UndoRedo'
+import { replaceUndoStack } from '../../store/actions/undo'
+
+// `getRemixOnFirstEdit` is a legacy function, so for the purposes of this
+// test suite it always returns `false` (aka, assume user owns the street)
+jest.mock('../../streets/remix', () => ({
+  getRemixOnFirstEdit: () => false
 }))
 
 describe('UndoRedo', () => {
-  it('renders two buttons', () => {
-    const wrapper = mount(<UndoRedo locale={{}} />)
-    expect(wrapper.find('button').length).toEqual(2)
+  afterEach(() => {
+    cleanup()
   })
 
-  it('checks if undo or redo is available when undo position changes', () => {
-    // `<UndoRedo>` is mounted with initial props
-    const wrapper = mount(<UndoRedo undoPosition={0} undoStack={[]} />)
+  it('renders two buttons', () => {
```

Comment:
I'll keep it here because the snapshots are handy to have, but I'll leave a todo note to remove it once we have a snapshot on the outer palette container.